docs/monitors/df.md

```diff
@@ -0,0 +1,79 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# df
+
+
+This monitor reports metrics about free disk space.
```

Comment:
Does the gopsutil library require that the disks be mounted to collect stats on them?  If so, document it here since this is sometimes a point of confusion, esp when using containers.